src/Sylius/Bundle/PricingBundle/Form/EventListener/BuildPriceableFormSubscriber.php

```diff
@@ -85,8 +86,13 @@ public function preBind(FormEvent $event)
      * @param string        $calculatorType
      * @param array         $data
      */
-    protected function addConfigurationFields(FormInterface $form, $calculatorType, array $data = array())
+    protected function addConfigurationFields(FormInterface $form, $calculatorType = null, array $data = array())
```

Comment:
This is a good thing to add but `$caculatorType` sometimes will passed as null (e.g. on [line 79](https://github.com/Sylius/Sylius/pull/3061#discussion-diff-38894690R79)), so that will override argument default value behavior.